src/common/debugman.h

```diff
@@ -56,9 +56,11 @@ enum DebugChannel {
 	kDebugEngineSound   , ///< "ESound", engine sound.
 	kDebugEngineVideo   , ///< "EVideo", engine video.
 	kDebugEngineEvents  , ///< "EEvents", engine events.
-	kDebugEngineScripts , ///< "EScripts", engine scripts.
 	kDebugEngineLogic   , ///< "ELogic", engine game logic.
 
+	kDebugEngineScripts     , ///< "EScripts", engine nwscripts
+	kDebugEngineActionScript, ///< "EActionScript", engine actionscript
```

Comment:
kDebugEngineActionScript is the wrong debug channel for the action script there.

If you look into NWScript, it uses kDebugScripts for the general script disassembly, things that are the same for every engine. kDebugEngineScripts is for things that the engine modifies or does, like engine functions.

I.e. you should rather use kDebugActionScript for most (all?) of the debug stuff in the ActionScript interpreter there.